docs/ref/contrib/admin/index.txt

```diff
@@ -5,11 +5,15 @@ The Django admin site
 .. module:: django.contrib.admin
    :synopsis: Django's admin site.
 
-One of the most powerful parts of Django is the automatic admin interface. It
-reads metadata in your model to provide a powerful and production-ready
-interface that content producers can immediately use to start adding content to
-the site. In this document, we discuss how to activate, use and customize
-Django's admin interface.
+One of the most powerful parts of Django is the automatic admin interface.
+
+The admin reads metadata in your model to provide a quick and dirty interface
+that trusted users can use to start adding content to the site. It's often used
+as an organization's internal tool to manage a site. Don't try to build your
+entire site in the admin!
```

Comment:
Maybe "It's **highly** recommended to use Django admin site only as an administrative tool for managing content. It's not intended for building frontend." or something like that? Just tried to tone down last sentence a little bit.